Source/DafnyCore/AST/Types/Types.cs

```diff
@@ -2371,7 +2371,7 @@ public UserDefinedType(TypeParameter tp)
 
   /// <summary>
   /// This constructor constructs a resolved type parameter (but shouldn't be called if "tp" denotes
-  /// the .TheType of an opaque type -- use the (OpaqueType_AsParameter, OpaqueTypeDecl, List(Type))
+  /// the .TheType of an abstract type -- use the (AbstractType_AsParameter, AbstractTypeDecl, List(Type))
```

Comment:
This entire parenthetic remark is stale and can be deleted. (The `.TheType` field and `OpaqueType_AsParameter` thing went away several years ago.)

Reply:
Edited